src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -24,6 +24,7 @@ public static partial class PlatformDetection
         public static bool IsMonoInterpreter => GetIsRunningOnMonoInterpreter();
         public static bool IsFreeBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD"));
         public static bool IsNetBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
+        public static bool IsIOS => RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS"));
```

Comment:
nit, we use `iOS` instead of `IOS`: https://github.com/dotnet/runtime/blob/54e08b663fd4343e194d9162d7792566b23d380d/src/mono/Directory.Build.props#L40. cc @akoeplinger

Reply:
is msbuild case sensitive or not? `IOS` is what RuntimeInformation returns AFAIK. This is consistent with FREEBSD here and FreeBSD in project files.

Reply:
> is msbuild case sensitive or not?

It's not.

Reply:
`OSPlatform.Create("IOS"))` is correct here since the OSPlatform string-based API requires uppercase.

Reply:
oh you were probably talking about the property name. In that case the convention would be `IsiOS` even though it looks a bit weird.

Reply:
Alternatively IsIos? to match .NET convention (ignoring Apple)

Reply:
that looks even uglier :) we've used Apple's name everywhere else.

Reply:
BTW I did fix iOS but ignoring iWatch and  tvOS. What is returned for them @akoeplinger as OSPlatform?

Reply:
looks like we commented at the same time, let's add tvOS in this PR (we've been holding off on watchOS for now).

Reply:
should that be RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS") ?
It feels like there will be lot of similarities among the three (and maybe OSX).
Would it be worth of adding some Is.... to describe them all?

Reply:
yes it's `OSPlatform.Create("TVOS")`.

the platforms are not totally the same so we've been using `ios || tvos` checks instead since it's easier to refactor later on when everything is brought up and we know what's really the same.

Reply:
added. I'm not sue if there is good casing to to look right. One more I can think is is IsIOs and IsTvOs. still looks weird to apple users. single 'i' makes it hard.

Reply:
the proper casing would be `IsiOS` and `IstvOS` since that is what we use in other places and I'd prefer we use it consistently. If you feel strongly and want to match some .NET convention I won't block it though since this is test code :)

Reply:
no, I don't have string preference. It is combination with `Is` what makes it more difficult. 
With this, at least it is same as names used for msbuild.  Since that is what @ViktorHofer wanted, I think there is quorum.